qa/TL1_custom_cpp_lib/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+do_once() {
+  apt-get update
+  apt-get -y install cmake
```

Comment:
why do you need this here? I don't see this in TL1_git_submodule.

Reply:
Actually, I went further and need to build cmake from scratch in this script (cause in ubuntu package there's 3.10 version and we need 3.13)